docker-compose.yml

```diff
@@ -1,48 +1,52 @@
 version: '2'
-services: 
-    xnat-web:
-       build: ./tomcat
-       ports:
-        - "8081:8080"
-       volumes:
-         - ./webapps:/opt/tomcat/webapps
-       expose:
-        - "8080"
-       links:
-        - xnat-db    
-    xnat-db:
-       build: ./postgres
-       expose:
-         - "5432"
-       volumes:
-         - ./postgres-data:/var/lib/postgresql/data
-    xnat-nginx:
-       build: ./nginx
-       ports:
-        - "80:80"
-       expose: 
-        - "80"
-       links:
-         - xnat-web
+services:
+  xnat-web:
+    build: ./xnat17
+    ports:
+      - "8081:8080"
+    volumes:
+      - ./xnat-home:/data/xnat
+    links:
+      - xnat-db
+    environment:
+      - XNAT_DATASOURCE_DRIVER=org.postgresql.Driver
+      - XNAT_DATASOURCE_URL=jdbc:postgresql://xnat-db/xnat
+      - XNAT_DATASOURCE_USERNAME=xnat
```

Comment:
This isn't your problem to solve here, but I don't like having the database username and password written here and also in `postgres/XNAT.sql`. We should be able to write the value in one variable or something in `docker-compose.yaml` and generate all the references using that variable.

I think I may file an issue for that after this PR is merged.